packages/SwingSet/docs/timer.md

```diff
@@ -33,29 +33,31 @@ and creating the service. The vat is responsible for creating repeater objects
 to wrap the device node as a capability that can be accessed by regular vat
 code.
 
-In `bootstrap()` for a particular SwingSet, we create a timerService, and
-make it accessible to the user in `home`
+In `bootstrap()` for a particular SwingSet, we create a timerService, and make
+it accessible to the user in `home.localTimerService` (whose timestamps are
+measured in milliseconds).
 
 ```
 const timerService = E(vats.timerWrapper).createTimerService(devices.timer);
 ```
 
-Then users in the REPL can use the timerService to schedule wakeups.
+Then users in the REPL can use the `localTimerService` or `chainTimerService`
+(both in seconds) to schedule wakeups.
```

Comment:
This seems inconsistent with the above. Is `localTimerService` in seconds or milliseconds?

Reply:
Used to be seconds (but was incorrect).  Now I've moved to milliseconds and corrected the polling to be accurate.